apps/aecore/src/aec_consensus_hc.erl

```diff
@@ -99,8 +101,16 @@ start(Config, #{block_production := BlockProduction}) ->
             <<"polling">> :=
                 #{  <<"fetch_interval">> := FetchInterval,
                     <<"nodes">> := Nodes0
-                 } = Polling
-          }} = Config,
+                 } = Polling,
+            <<"producing_commitments">> := ProducingCommitments
+          },
+     <<"time_till_declaring_lazy_leader">> := _TimeTillDeclaringLazy
```

Comment:
Not sure about the name here. "time_until_lazy_leader" maybe, or "lazy_leader_trigger_time"